src/AboutDialog.cpp

```diff
@@ -146,12 +146,12 @@ void AboutDialog::CreateCreditsList()
    AddCredit(wxT("AndroidKitKat"), sneederFormat, roleSneed);
    AddCredit(wxT("moxniso"), sneederFormat, roleSneed);
    AddCredit(wxT("kakolisgay"), sneederFormat, roleSneed);
-   AddCredit(wxT("joabbautista9"), sneederFormat, roleSneed);
    AddCredit(wxT("IThinkImOKAY"), sneederFormat, roleSneed);
    AddCredit(wxT("Furitenman"), sneederFormat, roleSneed);
    AddCredit(wxT("baboyl"), sneederFormat, roleSneed);
    AddCredit(wxT("ishimotoshinobu"), sneederFormat, roleSneed);
    AddCredit(wxT("sowa705"), sneederFormat, roleSneed);
+   
```

Comment:
are you sure you want this blank line in

Reply:
anonymous posters use a different formatter, they should be separated